web/datacatalog/datastore.py

```diff
@@ -46,17 +46,16 @@ def __init__(self):
 
 async def is_healthy():
     for datastore in implemented_datastores:
-        is_healthy = await datastore.is_healthy()
-        if not is_healthy:
+        if not await datastore.is_healthy():
             return False
     return True
 
 
 async def get_by_id(id):
     for datastore in implemented_datastores:
-        object = await datastore.get_by_id(id)
-        if object:
-            return object
```

Comment:
`object` is een python built-in. Ik probeer het hergebruik van built-in identifiers te vermijden.